polymetis/scripts/mirror_sim.py

```diff
@@ -61,19 +65,23 @@ def main(cfg):
         ignore_gravity=hw_robot.use_grav_comp,
     )
 
-    # Reset and test in sim, TODO: not easily doable until the mirror sim is moved into RoboInterface
-    # sim_robot.go_home()
-    # sim_robot.send_torch_policy(policy)
+    print("Setting up mirror for forward...")
+    hw_robot.setup_mirror_for_forward()
+    print("Homing simulation...")
+    hw_robot.go_home(use_mirror=True)
+    print("Running forward sim...")
+    hw_robot.send_torch_policy(policy, use_mirror=True)
+    hw_robot.clean_mirror_after_forward()
 
     # Mirror
+    print("Syncing...")
+    hw_robot.sync_with_mirror()
     print("Homing robot...")
     hw_robot.go_home()
-    print("Syncing...")
-    mirror_sim_client.sync(hw_robot)  # must be non-blocking
     print("Sending policy...")
     hw_robot.send_torch_policy(policy)
     print("Unsyncing...")
```

Comment:
This still seems quite complicated from the user side.
Can the user simply do 
```py
robot.go_home()   # mirror_sim also goes home automatically because they're in sync
robot.send_torch_policy(policy, use_mirror=True)
robot.send_torch_policy(policy)
```

Reply:
Defaulting to mirror mode and switching to forward mode only when running a policy on the mirror sim would enable this simpler UI. I think only blocking send policy calls would be supported to prevent concurrently mirroring and running a forward policy. Does this sound good @exhaustin?

Reply:
I think this makes sense. I don't think there are any users that are using the non-blocking version of the move_to calls, @1heart can you confirm?

Reply:
What about everyone using the `start_*_impedance`, `update_desired_*_positions` methods? Those are non-blocking.

Maybe a cleaner way to do this is with the `with` paradigm:


```
with hw_robot.viz_mirror():
   hw_robot.go_home()  # hw reflected in sim

with hw_robot.viz_sim():  # or some other name for this
    hw_robot.send_torch_policy(policy)  # only reflected in sim
```